docs/shared/recipes/console-troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+# Nx Console Troubleshooting
+
+## VSCode + nvm Issues
+
+VSCode loads a version of Node when it starts. It can use versions set via [`nvm`](https://github.com/nvm-sh/nvm) but there's some caveats.
```

Comment:
```suggestion
VSCode loads a version of Node when it starts. It can use versions set via [`nvm`](https://github.com/nvm-sh/nvm) but there are some caveats.
```